packages/core/src/Stores/Modules/Cashier/cashier-store.js

```diff
@@ -599,7 +607,18 @@ export default class CashierStore extends BaseStore {
 
         if (response_verify_email.error) {
             this.clearVerification();
-            this.setErrorMessage(response_verify_email.error);
+            if (response_verify_email.msg_type === 'verify_email') {
```

Comment:
I think for all verify email errors we want to set verification error, is there a need for this if/else?

Reply:
I don't think we need it since the outer `if/else` handles email verification errors 👍🏻